docs/100_producten/100_borgstelling/borgstelling-aanvragen.feature

```diff
@@ -33,7 +33,7 @@ Functionaliteit: Borgstelling aanvragen
     En het Schuldenknooppunt bericht is verwerkt
     Dan is de status van de borgstelling "afgegeven"
     En is "het contract" gearchiveerd
-    En is er een verkoop geregistreerd van € "50.0"
+    En is er een verkoop geregistreerd van € "49.99"
```

Comment:
```suggestion
    En is er een verkoop geregistreerd van € 49,99
```
Klopt de afronding strategie? @Frank-SBN 1% van 4999,99: hoe moet dat op twee decimalen afgerond worden?

Reply:
@arjangeertsema @Frank-SBN Op dit moment sturen wij het bedrag als letterlijk 1%, dus in dit geval sturen wij naar exact 49.9999. Ik weet niet precies hoe/en/of Exact dit afrond. Mocht hier nog een wijziging nodig zijn.